CHANGELOG.md

```diff
@@ -3,9 +3,89 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.0
+
+_Aug 23, 2021_
+
+This is the first offitial stable release of the `DataGrid` and `DataGridPro`. It supports `@material-ui/core` v4 and has a partial support for v5. From this release on we will release new features for the `DataGrid` and `DataGridPro` under version `5.0.0-beta.0` and have full support for `@material-ui/core` v5.
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Introduce the [row editing](https://material-ui.com/components/data-grid/editing/#row-editing) feature (#2098) @m4theushw
+- ⚛️ Remove unused row CSS classes (#2327) @ZeeshanTamboli
+- ⚡️ Rename `XGrid` to `DataGridPro` (#2382) @m4theushw
+- ✨ Move packages to `@mui` scope (#2341) @oliviertassinari
+- 💡 The `api` property was removed from the callback params. To access the API, use the `DataGridPro` (#2312) @DanailH
+
+This is the last alpha release. We are moving to beta in the next release, next week.
+
+### `@mui/x-data-grid@v4.0.0-alpha.38` / `@mui/x-data-grid-pro@v4.0.0-alpha.38`
+
+#### Breaking changes
+
+- [DataGrid] Remove unused row CSS classes (#2327) @ZeeshanTamboli
+  The CSS classes `.Mui-odd` and `.Mui-even` were removed from the row.
+
+- [DataGridPro] Rename `XGrid` to `DataGridPro` (#2382) @m4theushw
+
+  ```diff
+  -import { XGrid } from '@material-ui/x-grid';
+  +import { DataGridPro } from '@mui/x-data-grid-pro';
+  ```
+
+- [core] Move packages to `@mui` scope (#2341) @oliviertassinari
```

Comment:
I think that we could group these two changes, they are in the end, a single migration step for developers.